### DISCUSSION TITLE
A short alternative of starts keyword

### DISCUSSION THREAD

**Discussion by AniketKumar2500** [CsqLang/Csq3#17] [Ideas]
When i was typing starts keyword it consumes some time so will you have any idea what keyword could be a short alternative for starts?

**Comment by AniketKumar2500** [comment]
What do you say about?
def df():
    ....
df ends

if condition:
    ....
ends

for expression:
    ....
ends

class c:
    ....
ends

**Reply by WithSJ** [reply]
hmmm...
 that's good

**Reply by AniketKumar2500** [reply]
So shall i write code for parser?

**Reply by WithSJ** [reply]
yup... 
 make it easier :)

**Comment by AniketKumar2500** [comment]
What about?
def df()
....
df ends

if condition
....
ends

for expression
....
ends

class c
....
ends
